plumbing/object/commit.go

```diff
@@ -222,6 +222,32 @@ func (b *Commit) Encode(o plumbing.EncodedObject) error {
 	return err
 }
 
+// Stats shows the status of commit.
+func (c *Commit) Stats() (FileStats, error) {
+	// Get the previous commit.
+	ci := c.Parents()
+	parentCommit, err := ci.Next()
+	if err != nil {
+		if err == io.EOF {
+			emptyNoder := treeNoder{}
+			parentCommit = &Commit{
+				Hash: emptyNoder.hash,
+				// TreeHash: emptyNoder.parent.Hash,
```

Comment:
@mcuadros I think I'm not using it the way you meant.
This is for the first commit scenario, no parent commit.
Without assigning a `TreeHash` I get `panic: object not found`.
And if I uncomment the above line, it panics with `invalid memory address` error.

Missing something really simple? 😅